interactions/api/models/message.pyi

```diff
@@ -51,6 +54,8 @@ class ChannelMention(DictSerializerMixin):
     def __init__(self, **kwargs): ...
 
 class Message(DictSerializerMixin):
+    client: Optional[HTTPClient]
+    http: HTTPClient
```

Comment:
Why do you have two attributes of the same thing with one being optional?

Reply:
I was testing something and forgot to undo it, will change